plugin/luasnip.vim

```diff
@@ -14,6 +14,10 @@ function! luasnip#expand_or_jumpable()
 	return luaeval('require("luasnip").expand_or_jumpable()')
 endfunction
 
+function! luasnip#jumpable(direction)
+	return luaeval('require("luasnip").expand_or_jumpable(_A)', a:direction)
```

Comment:
I don't quite get why this call goes to `expand_or_jumpable` instead of `jumpable`